mmcv/runner/hooks/logger/wandb.py

```diff
@@ -14,13 +17,28 @@ def __init__(self,
                  reset_flag=False,
                  commit=True,
                  by_epoch=True,
-                 with_step=True):
+                 with_step=True,
+                 config_path=None):
+        
         super(WandbLoggerHook, self).__init__(interval, ignore_last,
                                               reset_flag, by_epoch)
+        
+        """
+        Args:
+            
+            with_step (bool): whether making a log in each step or not.
+                        Default: True.
+            config_path (str, optional): The path of final config of each experiment.
+                        It can be either a path of final config file or a directory of 
+                        config files. The config is uploaded to wandb server if it is
+                        not None. Default: None.
+        """
```

Comment:
```suggestion
        """
        Args:
            with_step (bool): whether making a log in each step or not.
                Default: True.
            config_path (str, optional): The path of the final config of each
                experiment. It can be either a path of the final config file or
                a directory of config files. The config is uploaded to wandb
                server if it is not None. Default: None.
                `New in version 1.3.18.` 
        """
```

Reply:
Thank you for the correction.